src/main/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollectionAdapter.java

```diff
@@ -219,7 +219,7 @@ private void checkArguments(Object[] fieldsAndValues) {
             if (field == null || field.isEmpty()) {
                 throw new IllegalArgumentException(INVALID_FIELD_MESSAGE);
             }
-            query.whereEqualTo(field, value);
+            query = query.whereEqualTo(field, value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment of parameter 'query' is not allowed.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)